apps/stats/tailwind.config.js

```diff
@@ -1,6 +1,7 @@
 const { join } = require('path');
 const { createGlobPatternsForDependencies } = require('@nrwl/next/tailwind');
 const theme = require('../../libs/tailwindcss-config/src/theme');
+const vegaCustomClasses = require('../../libs/tailwindcss-config/src/vega-custom-classes');
```

Comment:
question (non-blocking): is there a reason we can't import fom the package and need to use relative imports? Shouldn't everything needed to use the package be exported from it 🤔 

Happy to do this as a follow up.

Reply:
Good point!